File: ezSCORES/UI/ezscores_mobile/lib/models/applications.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:ezscores_mobile/models/teams.dart';
import 'package:ezscores_mobile/models/competitions.dart';
part 'applications.g.dart';

@JsonSerializable()
class Applications {
	int? id;
	int? teamId;
	int? competitionId;
	String? message;
	bool? isPaId;
  double? paidAmount;
	bool? isAccepted;
  Teams? team;
  Competitions? competition;

	Applications();

	factory Applications.fromJson(Map<String, dynamic> json) => _$ApplicationsFromJson(json);

	Map<String, dynamic> toJson() => _$ApplicationsToJson(this);
}

File: ezSCORES/ezSCORES.Model/Applications.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class Applications
	{
		public int Id { get; set; }

		public int TeamId { get; set; }

		public int CompetitionId { get; set; }

		public string? Message { get; set; }

		public bool IsPaId { get; set; }

		public float? PaIdAmount { get; set; }

		public bool? IsAccepted { get; set; }
		public virtual Teams Team { get; set; } = null!;
		public virtual Competitions Competition { get; set; } = null!;
	}
}
